scripts/common.sh

```diff
@@ -45,7 +45,7 @@ GCLOUD_CLI_CONTAINER_IMAGE_ID="gcr.io/google.com/cloudsdktool/cloud-sdk:397.0.0"
 # shellcheck disable=SC2034
 TERRAFORM_CONTAINER_IMAGE_ID="hashicorp/terraform:1.1.3"
 # shellcheck disable=SC2034
-OS_INSTALLER_IMAGE_URL="https://releases.ubuntu.com/focal/ubuntu-20.04.5-live-server-amd64.iso"
+OS_INSTALLER_IMAGE_URL="https://releases.ubuntu.com/focal/ubuntu-20.04.6-live-server-amd64.iso"
```

Comment:
1. Can you add a step in the [validate shell scripts](https://github.com/GoogleCloudPlatform/solutions-viai-edge-provisioning-configuration/blob/main/.github/workflows/validate-shell-scripts.yml) job to run the `download-os-images.sh` script, so that we ensure these URLs are valid?
2. Can you add a trigger to the `validate shell scripts` job to also [run on a schedule](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule), so we periodically validate the scripts, and not just on PRs and commits? This would help proactively catch issues with URLs.

If you need any help with the above, please let me know.

Thanks!

Reply:
1. Add weekly cron job to validate all scripts
2. Validate `download-os-images.sh`